dns/operationcode.org/records.tf

```diff
@@ -17,4 +17,18 @@ resource "dnsimple_record" "dashboards" {
   name = "dashboards"
   type = "CNAME"
   value = "${var.k8s-cluster-ingress}"
-}
\ No newline at end of file
+}
+
+resource "dnsimple_record" "resources_api" {
+  domain = "${var.hosted-zone}"
+  name = "resources"
+  type = "CNAME"
+  value = "${var.k8s-cluster-ingress}"
+}
+
+resource "dnsimple_record" "resources_staging_api" {
+  domain = "${var.hosted-zone}"
+  name = "resources.staging"
```

Comment:
this is `resources.staging.operationcode.org` which is similar to `api.staging.operationcode.org` ?

Reply:
Yes, that's for the staging environment